src/index.js

```diff
@@ -248,6 +248,10 @@ export default class RNPickerSelect extends PureComponent {
             Keyboard.dismiss();
         }
 
+        if (this.pickerRef && Platform.OS == 'android') {
+            this.pickerRef.focus();
+        };
```

Comment:
Should take into account whether picker is currently visible or not.

```suggestion
        if (this.pickerRef && Platform.OS === 'android') {
          if (!showPicker) {
            this.pickerRef.focus();
          } else {
            this.pickerRef.blur();
          }
        }
```

Reply:
FYI, this didn't work, using only `this.pickerRef.focus()` did.